tests/unit/Annotations/Annotation/GetExpressionCest.php

```diff
@@ -13,20 +13,51 @@
 
 namespace Phalcon\Test\Unit\Annotations\Annotation;
 
+use Phalcon\Annotations\Annotation;
 use UnitTester;
 
 class GetExpressionCest
 {
+    private $PHANNOT_T_STRING = 303;
+
     /**
      * Tests Phalcon\Annotations\Annotation :: getExpression()
      *
-     * @author Phalcon Team <team@phalcon.io>
-     * @since  2018-11-13
+     * @author Jeremy PASTOURET <https://github.com/jenovateurs>
+     * @since  2020-01-22
      */
     public function annotationsAnnotationGetExpression(UnitTester $I)
     {
         $I->wantToTest('Annotations\Annotation - getExpression()');
 
-        $I->skipTest('Need implementation');
+        $sValue = 'test';
+        $sValue1 = 'test1';
+
+        $aOneExpr = [
+            'type'  => $this->PHANNOT_T_STRING,
```

Comment:
Something in me says we should change `$this->PHANNOT_T_STRING` to `self::PHANNOT_T_STRING` but remember why. I think it was something performance wise but can't remember. @niden ideas?

Reply:
I have two problems about constants:
- Currently, I have copied/pasted the same constants in a few tests of Annotation
- When I try to use constants in the class, PHP_CodeSniffer triggers an error because for PHP > 7.1, we need to specify if it is a private/public/protected constant.